hackclub.com.yaml

```
@@ -1618,6 +1623,18 @@ mangu:
   - ttl: 600
     type: CNAME
     value: mangu.pages.dev.
+  - ttl: 600
+    type: MX
+    values: 
+      - exchange:  mx1.forwardemail.net
+        preference: 10
+      - exchange:  mx2.forwardemail.net
+        preference: 10
+  - ttl: 5
+    type: TXT
+    values:
+      - forward-email=johnkelsey.irungu@gmail.com
+       
```

```suggestion
  - ttl: 600
    type: MX
    values: 
      - exchange:  mx1.forwardemail.net.
        preference: 10
      - exchange:  mx2.forwardemail.net.
        preference: 10
  - ttl: 600
    type: TXT
    values:
      - forward-email=johnkelsey.irungu@gmail.com
       
```

This just modifies the TXT record's TTL and adds the required trailing dot to the exchanges